src/components/GlobalPreferences/CustomLabels/LocalIdentities.js

```diff
@@ -287,11 +290,17 @@ function Actions({ onExport, onRemove, onShare, someSelected }) {
 
   return (
     <React.Fragment>
-      <ButtonDropDown
+      <DropDown
+        disabled={disabled}
         compact={compact}
+        selected={-1}
         css="z-index: 2;"
```

Comment:
Do we still need the z-indexes on these components (there are a couple defined in this file)?

Reply:
Yes, this are `z-index`es for the `Modal`s (`Share` and `Remove`) which use portals to show at root level.

Reply:
But we're also declaring the z-indexes on the DropDown and its placeholder, which seems a bit weird and are not related to the modals?

Reply:
True, good eye, those were inherited from the local `ButtonDropDown` that was not using `Popover`, I'll remove them.